docs/source/pages/supported_technologies/index.rst

```diff
@@ -107,6 +107,17 @@ Provenances
        * The provenance should be published on JFrog Artifactory
      - :doc:`page </pages/supported_technologies/jfrog>`
 
+.. _supported_automatic_deps_resolution:
+
+-------------------------------
+Automatic dependency resolution
+-------------------------------
+
+Currently, we support the following type of project for automatic dependency resolution.
+
+* Java Maven
+* Java gradle
```

Comment:
Inconsistent capitalisation: `gradle` -> `Gradle`

Reply:
Fixed in https://github.com/oracle/macaron/pull/840/commits/92efd8d5b38ba1cb63d88cd6549e6bac10033178